static/js/tools/choropleth/choropleth.tsx

```diff
@@ -105,23 +137,42 @@ class ChoroplethMap extends Component {
           "statistical variable choice! Please select a new variable.";
       }
     );
-  }
+  };
+
+
+/**
+ * Gets the min or max date in a list of ISO dates.
+ * @param: dates: a list of dates in ISO format. Example: "2020-01-02".
+ * @param: minMax: whether to return "min" or "max" of dates.
+ * Default is "max".
+ */
+getMinMaxDate = (dates: string[], minMax?: 'min' | 'max'): string => {
```

Comment:
The only reason I added min was because, I was thinking maybe in the future we wanted to work with ranges of dates.
But for now, it's not needed. If we ever need it, we can simply add it again.
Fixed.